documentation/host/deployment.rst

```diff
@@ -48,7 +48,7 @@ Keep in mind that FlexMeasures is based on `Flask <https://flask.palletsprojects
 Install the linear solver on the server
 ---------------------------------------
 
-To compute schedules, FlexMeasures uses the `Cbc <https://github.com/coin-or/Cbc>`_ mixed integer linear optimization solver.
+To compute schedules, FlexMeasures uses the `Cbc <https://github.com/coin-or/Cbc>`_ or `HiGHS <https://highs.dev/>`_ mixed integer linear optimization solvers.
```

Comment:
- Let's mention that (for now) Cbc is the default. 
- I don't believe the extra "s" after "solver" is needed.
- The next sentence starts with "It" but it's not clear what is being referenced.